aspnetcore/fundamentals/target-aspnetcore.md

```diff
@@ -0,0 +1,213 @@
+
+If targeting .NET Standard, add a package reference to [Microsoft.AspNetCore.Mvc.ViewFeatures](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.ViewFeatures). The `Microsoft.AspNetCore.Mvc.ViewFeatures` package moved into the shared framework and is therefore no longer published. For example:
+
+[!code-xml[](target-aspnetcore/samples/single-tfm/netstandard2.0-view-components-library.csproj)]
+
+## Support multiple ASP.NET Core versions
+
+Multi-targeting is required to author a library that supports multiple variants of ASP.NET Core. Consider a scenario in which a Tag Helpers library must support the following ASP.NET Core variants:
+
+* ASP.NET Core 2.1 targeting .NET Framework 4.6.1
+* ASP.NET Core 2.x targeting .NET Core 2.x
+* ASP.NET Core 3.0 targeting .NET Core 3.0
+
+The following project file supports these variants via the `TargetFrameworks` property:
+
+[!code-xml[](target-aspnetcore/samples/multi-tfm/recommended-tag-helpers-library.csproj)]
+
+With the preceding project file:
+
+* The `Markdig` package is added for all consumers.
+* A reference to [Microsoft.AspNetCore.Mvc.Razor](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.Razor) is added for consumers targeting .NET Framework 4.6.1 or .NET Core 2.x. Version 2.1.0 of the package works with ASP.NET Core 2.2 because of backwards compatibility.
+* The shared framework is referenced for consumers targeting .NET Core 3.0. The `Microsoft.AspNetCore.Mvc.Razor` package is included in the shared framework.
+
+Alternatively, .NET Standard 2.0 could be targeted instead of targeting both .NET Core 2.1 and .NET Framework 4.6.1:
+
+[!code-xml[](target-aspnetcore/samples/multi-tfm/alternative-tag-helpers-library.csproj?highlight=4)]
+
+With the preceding project file, .NET Core 2.x and .NET Framework 4.6.1 projects are supported because both target frameworks implement .NET Standard 2.0. Since this library only contains Tag Helpers, it's more efficient to target the specific platforms on which ASP.NET Core runs: .NET Core and .NET Framework. Tag Helpers can't be used by other .NET Standard 2.0-compliant target frameworks such as Unity, UWP, and Xamarin. If your library needs to call platform-specific APIs, target specific .NET implementations instead of .NET Standard. For more information, see [Multi-targeting](/dotnet/standard/library-guidance/cross-platform-targeting#multi-targeting).
+
+## Use an API that hasn't changed
+
+Imagine a scenario in which you're upgrading a middleware library from .NET Core 2.2 to 3.0. The ASP.NET Core middleware APIs being used in the library haven't changed between ASP.NET Core 2.2 and 3.0. To continue supporting the middleware library in .NET Core 3.0, take the following steps:
+
+* Follow the [standard library guidance](/dotnet/standard/library-guidance/).
+* Add a package reference for the API's NuGet package if the assembly doesn't exist in the shared framework.
```

Comment:
We should talk about what **version** of the package to reference whenever we say this, or have a section that can be referred to that does.

Reply:
Do you have a specific example in mind that I could show in this section?

Reply:
And now I think about it more, his might just be **too complicated** to give general guidance on 😕  Let's ignore my original comment for now.

Reply:
Sounds good. If you change your mind, let me know. I'm sure we'll get some good feedback from the community as well. This doc would be worth sharing in the next community standup.